en/activity-standard/iati-activities/iati-activity/result/indicator/period.rst

```diff
@@ -6,6 +6,5 @@ Example usage of ``period`` in context of an ``indicator`` in a ``result`` eleme
 
 .. literalinclude:: ../../../../activity-standard-example-annotated.xml
 	:language: xml
-	:start-after: <!--result starts-->
-	:end-before: <!--result ends-->
-
+	:start-after: <!--result-indicator starts-->
```

Comment:
Since this is a page for the `period` element, should this be `result-period` instead?

Reply:
It has been noted that showing the context within the parent is done elsewhere, so it's fine as-is.